list.json

```diff
@@ -1,146 +1,286 @@
 {
 	"corpora": {
 		"quora-duplicate-questions": {
-			"description": "over 400,000 lines of potential question duplicate pairs. Each line contains IDs for each question in the pair, the full text for each question, and a binary value that indicates whether the line truly contains a duplicate pair.",
+			"num_rows": 404290,
+			"row_format": "dict",
+			"size": "21684784 (~21MB)",
```

Comment:
Exact number better in this structured field. The approximation can be either computed, or mentioned in the plain text description.

Reply:
Done  09b3098